CHANGELOG.md

```diff
@@ -14,12 +16,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     prevented AWS from halting/restarting warmed instances when task code was
     throwing consistent errors under load.
 
-- **CUMULUS-2200**
```

Comment:
This was moved to the release 3.0.0 CL as it should be, PR to master forthcoming.